webapp/sources/rudder/rudder-web/src/main/webapp/secure/configurationManager/techniqueEditor.html

```diff
@@ -89,23 +90,34 @@
   });
 
   app.ports.storeDraft.subscribe(function(draft) {
-    draft.date = new Date().toISOString();
-    var drafts = JSON.parse(localStorage.getItem("drafts"))
-    if (drafts === null) { drafts = {} }
-    drafts[draft.id] = draft;
-    localStorage.setItem("drafts", JSON.stringify(drafts));
+    try {
+      draft.date = new Date().toISOString();
+      var drafts = JSON.parse(localStorage.getItem("drafts"))
+      if (drafts === null) { drafts = {} }
+      drafts[draft.id] = draft;
+      localStorage.setItem("drafts", JSON.stringify(drafts));
+    } catch (error) {
+      createErrorNotification("An error occurred while saving technique draft, it will not be persisted in your browser.\nError details: " + error) ;
+    }
   });
 
   app.ports.clearDraft.subscribe(function(key) {
-    var drafts = JSON.parse(localStorage.getItem("drafts"));
-    delete drafts[key];
-    localStorage.setItem("drafts", JSON.stringify(drafts));
```

Comment:
why was it done and why do we remove that ?

Reply:
I added it back

Reply:
it was a mistake.

Reply:
ok